week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -7,8 +7,10 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
 
-function eligibleStudents() {
+function eligibleStudents(arr) {
+  let newArr = arr.filter((attendance) => attendance[1] >= 8);
 
+  return newArr.map((name) => name[0]);
 }
```

Comment:
I think you haven't updated your repository because we added an extra exercise to this as well as updated the description.